service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -34,13 +34,16 @@
 import java.util.Optional;
 import java.util.UUID;
 import javax.annotation.Nullable;
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 /** CRUD methods for controlled objects. */
 @Component
 public class ControlledResourceService {
 
+  private static final int MAX_ASSIGNED_USER_LENGTH = 128;
```

Comment:
After some reading, I believe the max size of an email is 254. Suggest we use that.
If you google it you get mountains of arguments saying it is 254, 255, 256, 320...